webpack/assets/javascripts/react_app/routes/Audits/AuditsPage/AuditsPageActions.js

```diff
@@ -19,21 +20,23 @@ import {
 import { stringifyParams, getParams } from '../../../common/urlHelpers';
 import { translate as __ } from '../../../common/I18n';
 
+const AUDITS_FULL_PATH = foremanUrl(AUDITS_PATH);
```

Comment:
Does `foremanUrl` work at compile time? I think it needs to be called at runtime.

Reply:
Oh yeah, this needs to be a method.